src/components/Pagination/Pagination.tsx

```diff
@@ -87,17 +87,17 @@ export const Pagination = ({
   const navClasses = classnames('usa-pagination', className)
 
   const isOnFirstPage = currentPage === 1
-  const isOnLastPage = currentPage === totalPages
+  const isOnLastPage = totalPages ? currentPage === totalPages : null
```

Comment:
Suggestion(nit-pick): Feels like the `null` here should be a `false` since this var is being used as a boolean.